ephypype/tests/test_preproc.py

```diff
@@ -29,9 +32,13 @@ def test_preprocess_fif():
     preprocess_fif(segment_raw_fname, l_freq=l_freq, h_freq=h_freq,
                    down_sfreq=down_sfreq)
 
+    os.chdir(current_wd)
```

Comment:
oh you still need to do this. Why don't you use what @dmalt suggested or the `_TempDir` from `mne` ? Better to use standard tools than reinventing the wheel, no?

Reply:
`_TempDir`  created a tmp dir that is used to create the path where to save the files.
Since we save using abs path we need to change the wd to this temp dir and it is what _create_tmp_dir do.
I removed `os.chdir(current_wd)`

Reply:
can you point me to the line where you use abs path? I thought if you just changed the path when creating your workflow, that would move your output files, no?

Reply:
> can you point me to the line where you use abs path? I thought if you just changed the path when creating your workflow, that would move your output files, no?

e.g. [here](https://github.com/neuropycon/ephypype/blob/74cc72da3462e465a155b9e2777c0c8796592e24/ephypype/preproc.py#L32)

we use abs path since the files are stored in the Nodes folder

Reply:
> hmm ... but this seems like a bad default. Shouldn't we be able to do [something like this](https://github.com/neuropycon/ephypype/blob/74cc72da3462e465a155b9e2777c0c8796592e24/examples/plot_power.py#L56) ?

no, it's not a bad default, we decided to save in the Node directory. @davidmeunier79 

In the Forward Node I don't save in the Node directory and it seems a problem...I have to inspect this issue now that I have to test the inverse pipeline

Reply:
Sorry not sure to follow the full conversation... but what about putting the full resulting workflow in some temporary dir (i.e.workflow.base_dir = temp_dir) when testing? 
So that everything from the test can be removed at the end. Would that solve the issue of abspath?

Reply:
Hi @davidmeunier79! about the testing now we are saving all in temp dir.

But I think @jasmainak is proposing to save the output node in a specified dir passed as input.
He doesn't like we save the output node using the abs path.

What I'm explaining to @jasmainak is that we decided to save the output of a Node in the Node directory and not e.g. in the subject dir.